files/gitignore.txt

```diff
@@ -9,6 +9,7 @@
 # Ignore all logfiles and tempfiles.
 /log/*.log
 /tmp
+.DS_Store
```

Comment:
@MProG let's not ignore it, because it's system specific. Look datarockets dotfiles